examples/dojo/js/todo/empty.js

```diff
@@ -0,0 +1 @@
+define(function () {});
```

Comment:
Added a comment to: https://github.com/asudoh/todomvc/blob/dojo110/examples/dojo/js/todo/empty.js#L1